doc/man1/openssl-req.pod.in

```diff
@@ -402,6 +403,11 @@ Non-interactive mode.
 
 Print extra details about the operations being performed.
 
+=item B<-quiet>
+
+Print fewer details about the operations being performed, which may be
+handy during batch scripts or pipelines.
```

Comment:
Maybe explicitly mention that keygen progress is disabled?

Reply:
Done